cpp/src/arrow/filesystem/azurefs_test.cc

```diff
@@ -889,45 +1096,6 @@ TEST_F(AzuriteFileSystemTest, DeleteDirSuccessHaveBlobs) {
   }
 }
 
-TEST_F(AzureHierarchicalNamespaceFileSystemTest, DeleteDirSuccessEmpty) {
```

Comment:
This is scary to look at, but I just re-ordered the tests: grouping them by environment.